tests/realtime/redisPublisher.js

```diff
@@ -81,6 +81,29 @@ describe('redis Publisher', () => {
       })
       .catch(done);
     });
+
+    it('when tried to publish sample without aspect,'+
```

Comment:
any other tests we can add here to make sure that we aren't breaking any of the existing realtime event publishing?